Unable to configure Recipe Robot preferences in Mojave

```
$ ./scripts/recipe-robot --config -v

                      -----------------------------------
                     |  Welcome to Recipe Robot v1.1.0.  |
                      -----------------------------------
                                \   _[]_
                                 \  [oo]
                                   d-||-b
                                     ||
                                   _/  \_
    
[ERROR] Recipe Robot exploded with unexpected error: RecipeTypes
Traceback (most recent call last):
  File "./scripts/recipe-robot", line 90, in main
    prefs = init_prefs(facts)
  File "./scripts/recipe-robot", line 273, in init_prefs
    if recipe["type"] in prefs["RecipeTypes"]:
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/PyObjC/objc/_convenience.py", line 162, in __getitem__objectForKey_
    return container_unwrap(res, KeyError, key)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/PyObjC/objc/_convenience.py", line 659, in container_unwrap
    raise exc_type(*exc_args)
KeyError: 'RecipeTypes'

[ERROR] 
```

--- a/scripts/recipe-robot
+++ b/scripts/recipe-robot
@@ -250,9 +250,9 @@ def init_prefs(facts):
     """Read Recipe Robot preferences.
 
     Preferences are handled in the following priority order:
-        0. If --config argument is specified, ignore prefs and rebuild.
-        3. If preferences plist doesn't exist, rebuild.
-        4. If preferences plist does exist, use it.
+        1. If --config argument is specified, ignore prefs and rebuild.
+        2. If preferences plist doesn't exist, rebuild.
+        3. If preferences plist does exist, use it.
 
     Args:
         facts: The Facts object with required keys:
@@ -267,13 +267,10 @@ def init_prefs(facts):
     recipes = facts["recipes"]
     # If prefs file exists, try to read from it.
     prefs = get_user_defaults()
-    if prefs:
+    if prefs and 'RecipeTypes' in prefs:
         for recipe in recipes:
             # Load preferred recipe types.
-            if recipe["type"] in prefs["RecipeTypes"]:
-                recipe["preferred"] = True
-            else:
-                recipe["preferred"] = False
+            recipe["preferred"] = recipe["type"] in prefs["RecipeTypes"]
 
         if args.config is True:
             robo_print("Showing configuration options...")

